### ISSUE TITLE
suggestion : est il prévu de faire des passerelles pour faire c...

### ISSUE THREAD

**Issue by systemDBugReport** [HUB-HUmanBeing/SystemD#108]
#### content :
est il prévu de faire des passerelles pour faire communiquer cet outil avec d'autres outils ?

**Comment by robinBanquo** [comment]
oui, pour toutes les parties "non encryptées" de system-d, cela sera possible de fournir une api pour que d'autre services puissent communiquer avec system-d.
apres pour ce qui est de la gestion de projet, nous souhaitons que tout les contenus restent chiffrés et cela risque de rendre impossible la communiquation avec d'autres outils (google calendar etc... )
